cmd/render.go

```diff
@@ -192,8 +196,12 @@ func Render() *cli.Command {
 				writer:  os.Stdout,
 				output:  c.String("output"),
 			}
+			ctx := make(map[string]any, 3)
```

Comment:
Why not create a struct instead? This would simplify the code and give us type safety